<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>444</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="headerTitle">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255); background-color: rgb(0, 255, 0);</string>
     </property>
     <property name="text">
      <string>REGISTRASI</string>
     </property>
     <property name="textFormat">
      <enum>Qt::TextFormat::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="IsiFrame">
     <property name="styleSheet">
      <string notr="true">.QFrame {
background-color: rgb(0, 33, 48);
}
QLineEdit, QTextEdit{
background-color: rgb(46, 47, 48);
}
QFrame{
font-weight: bold; color: #ffffff;}
QCheckBox::indicator:unchecked{background-color : rgb(46, 47, 48);}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <property name="spacing">
       <number>20</number>
      </property>
      <item row="3" column="0">
       <widget class="QLabel" name="emailLabel">
        <property name="text">
         <string>Email                  :</string>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QCheckBox" name="PerempuanCheckBox">
        <property name="text">
         <string>Perempuan</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">jeniskelaminButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="6" column="2" colspan="2">
       <widget class="QLineEdit" name="teleponInput"/>
      </item>
      <item row="4" column="2" colspan="2">
       <widget class="QLineEdit" name="passwordInput">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="passwordLabel">
        <property name="text">
         <string>Password          :</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="alamatLabel">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="tabletTracking">
         <bool>false</bool>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Alamat               :</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="teleponLabel">
        <property name="text">
         <string>Telepon             :</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="jeniskelaminLabel">
        <property name="text">
         <string>Jenis Kelamin   :</string>
        </property>
       </widget>
      </item>
      <item row="9" column="2" colspan="2">
       <widget class="QTextEdit" name="alamatInput">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>75</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QCheckBox" name="lakiCheckBox">
        <property name="text">
         <string>Laki-laki</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">jeniskelaminButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="namaLabel">
        <property name="text">
         <string>Nama Lengkap :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QLineEdit" name="namaInput"/>
      </item>
      <item row="3" column="2" colspan="2">
       <widget class="QLineEdit" name="emailInput"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="daftarFrame">
     <property name="styleSheet">
      <string notr="true">.QFrame {
background-color: rgb(0, 33, 48);
}
                QPushButton {
					background-color: rgb(0, 80, 115);
                    font-size: 20px;
                    font-weight: bold;
                    color: #ffffff;
                    padding-left: 20px;
                    border: none;
                    height: 50px;
                }
                QPushButton:checked {
                    background-color: rgb(0, 100, 200);
                    color: #ffffff;
                }
                QPushButton:hover {
                    background-color: rgb(0, 50, 100);
                }</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QPushButton" name="daftarButton">
        <property name="text">
         <string>DAFTAR</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="jeniskelaminButtonGroup"/>
 </buttongroups>
</ui>
